org.raceintospace.Raceintospace.yml

```diff
@@ -0,0 +1,89 @@
+# vim: sts=4:
+app-id: org.raceintospace.Raceintospace
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
```

Comment:
Update runtime to 20.08
```suggestion
runtime-version: '20.08'
```

Reply:
Oh, maybe tutorial should be updated. I just followed whatever was recommended there. Not really can find current platform to choose.